lib/serverless-tools/deployer.rb

```diff
@@ -5,12 +5,12 @@
 
 module ServerlessTools
   module Deployer
-    CONFIG_FILE = ENV["GITHUB_WORKSPACE"] ? "#{ENV["GITHUB_WORKSPACE"]}/functions.yml" : "functions.yml"
-
-    def self.deploy(action:, function: nil)
+    def self.deploy(action:, function: nil, filename: nil)
```

Comment:
Pretty sure this isn't needed and it was left over from when it was ran in the repo.